scale-tests/batch_test.py

```diff
@@ -53,8 +53,8 @@
     stream=sys.stdout)
 
 log = logging.getLogger(__name__)
-MONTE_CARLO_APP_URL = "http://xhuynh-dev.s3.amazonaws.com/monte-carlo-portfolio.py"
-GPU_IMAGE_RECOGNITION_APP_URL = "https://svt-dev.s3.amazonaws.com/run_image_recognition-final.py"
+MONTE_CARLO_APP_URL = "https://raw.githubusercontent.com/mesosphere/spark-build/master/scale-tests/apps/monte-carlo-portfolio.py"
+GPU_IMAGE_RECOGNITION_APP_URL = "https://raw.githubusercontent.com/mesosphere/spark-build/master/scale-tests/apps/image_recognition.py"
```

Comment:
@mpereira this is with assumption this PR gets merged to `master` prior to next scale test